internal/p2p/pex/reactor.go

```diff
@@ -317,10 +324,6 @@ func (r *Reactor) processPeerUpdate(peerUpdate p2p.PeerUpdate) {
 	}
 }
 
-func (r *Reactor) waitUntilNextRequest() <-chan time.Time {
-	return time.After(time.Until(r.nextRequestTime))
```

Comment:
Should we delete `nextRequestTime`? Looks like it's set in 'sendRequestForPeers` so it seems like this may be a mild behavior change.

Reply:
I believe that the only change here is that we're using (and reusing) a timer, rather than a bunch of time.Afters